portfolio/src/main/webapp/contact/scriptme.js

```diff
@@ -0,0 +1,55 @@
+/** Fetches tasks from the server and adds them to the DOM. */
+function loadTasks() {
+  console.log("lololl");
+  fetch('/list-contact').then(response => response.json()).then((contactsList) => {
+    const taskListElement = document.getElementById('contact-list');
+    contactsList.forEach((task) => {
+    taskListElement.appendChild(createTaskElement(task));
+    console.log(task);
+
+    })
+    console.log(contactsList);
+
+  });
+
+}
+
+/** Creates an element that represents a task, including its delete button. */
+function createTaskElement(task) {
+  const taskElement = document.createElement('li');
+  taskElement.className = 'task';
+
+  const nameElement = document.createElement('span');
+  const emailElement = document.createElement('span');
+  const phoneElement = document.createElement('span');
+  const messageElement = document.createElement('span');
+
+  nameElement.innerText = task.name;
+  emailElement.innerText = task.email;
+  phoneElement.innerText = task.phone;
+  messageElement.innerText = task.message;
+
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerText = 'Delete';
+  deleteButtonElement.addEventListener('click', () => {
+  deleteTask(task);
+
+    //Remove the task from the DOM. 
```

Comment:
Line this comment up with the line below. After `//`, make sure there is a space.

`// Remove the task from the DOM.`